moj_analytics/tests/conftest.py

```diff
@@ -0,0 +1,228 @@
+import json
+from unittest import mock
+
+import pytest
+from responses import RequestsMock
```

Comment:
These requirements are missing from `requirements.txt` - but maybe the `mojanalytics` module should be published to Pypi and added to `requirements.txt` instead

Reply:
For the time being I'd just say add the requirements.

Reply:
Also @andyhd I looked at the moj_analytics stuff and noticed it has **a lot** of pytest code.  I'd prefer if this was rewritten to standard django unit test and using mock/patch.  

A lot of it could be simplified or it indicates the client should be.